projects/plugins/crm/admin/dashboard/main.page.php

```diff
@@ -620,12 +620,13 @@ function jpcrm_render_dashboard_page() {
 				<tbody>
 					<?php
 					foreach ( $latest_cust as $cust ) {
-						// phpcs:disable WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase -- to be refactored.
-						$avatar = ( isset( $cust ) && isset( $cust['id'] ) ) ? $zbs->DAL->contacts->getContactAvatar( $cust['id'] ) : '';
-						$fname  = ( isset( $cust ) && isset( $cust['fname'] ) ) ? $cust['fname'] : '';
-						$lname  = ( isset( $cust ) && isset( $cust['lname'] ) ) ? $cust['lname'] : '';
-						$status = ( isset( $cust ) && isset( $cust['status'] ) ) ? $cust['status'] : '';
-						// phpcs:enable WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase
+						// phpcs:disable WordPress.NamingConventions.ValidVariableName -- to be refactored.
+						$contactAvatar = $zbs->DAL->contacts->getContactAvatar( $cust['id'] );
```

Comment:
I don't know why we are checking if `$cust['id']` is being set in the other lines, `$latest_cust` should only contain contacts with a valid id. But in any case, I will just leave a note here about this in case we get any warning messages in the future.